cabal.project

```diff
@@ -1,4 +1,4 @@
-index-state: 2021-12-07T00:00:00Z
+index-state: 2021-12-15T00:00:00Z
```

Comment:
I don't see why this bump is necessary. It's already much newer than that used in consensus.

Reply:
I reverted this change.